services/api/src/models/Quests.test.ts

```diff
@@ -1,34 +1,35 @@
-import {PUBLIC_PARTITION} from 'shared/schema/Constants';
-import {Quest} from 'shared/schema/Quests';
-import {QuestInstance} from './Database';
-import {searchQuests} from './Quests';
 import {
-  quests as q,
-  testingDBWithState,
-} from './TestData';
+  PUBLIC_PARTITION,
+  PRIVATE_PARTITION
+} from "../../../../shared/schema/Constants";
```

Comment:
We've generally styled things to use single quotes instead of double quotes, looks like you might need to update your editor / linter?